arwenmandos/gasSchedules/scripts/includetoml.go

```diff
@@ -0,0 +1,27 @@
+package main
```

Comment:
Consider moving this file to`cmd/includetoml/main.go`. All executable program should be in the same place. The file produced by `main()` uses a relative path anyway.

Reply:
It is only a macro, only used in the `go:generate` in its parent.

Reply:
`cmd` contains the public binaries